vme/zone/death.zon

```diff
@@ -936,6 +929,12 @@ code
    death_cry();
    make_corpse();
 
+   //Remove extras just Justice DB
```

Comment:
SHouldn't we remove all regardless of jurisiction. Not just midgaard and udgaard?